packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -259,7 +262,7 @@ test(testCase, async (t: Test) => {
         },
       };
       await apiClient.invokeContractV1(
```

Comment:
@aldousalvarez The original casting to `any` here was not necessary to begin with so you can avoid the entire ordeal by just doing this that also fixes the linter warning and makes it simpler to read:

```typescript
await apiClient.runTransactionV1(parameters);
```